internal/common/common.go

```diff
@@ -341,34 +366,73 @@ func GetEnv(key string, dfault string, combineWith ...string) string {
 	}
 }
 
+// Deprecated: use HostProcWithContext instead
```

Comment:
I'm not sure these changes warrant deprecating the existing methods since that could introduce lint issues for users unexpectedly and then require passing noop/background contexts for all usages.

Reply:
@shirou wdyt?

Reply:
Thank you for the suggestion. Yes, I agree about we should not add `Deprecated` at this time.

Reply:
ok removed.